packages/commerce-sdk-react/src/utils.ts

```diff
@@ -143,3 +144,20 @@ export const stringToBase64 =
     typeof window === 'object' && typeof window.document === 'object'
         ? btoa
         : (unencoded: string): string => Buffer.from(unencoded).toString('base64')
+
+/**
+ * Extracts custom parameters from a set of SCAPI parameters
+ *
+ * Custom parameters are identified by the 'c_' prefix before their key
+ *
+ * @param parameters object containing all parameters for a SCAPI / SLAS call
+ * @returns new object containing only custom parameters
+ */
+export const extractCustomParameters = (
+    parameters: {[s: string]: string | number | boolean | string[] | number[]} | null
+): helpers.CustomQueryParameters => {
+    if (typeof parameters !== 'object' || parameters === null) {
```

Comment:
Would `parameters` be string or number in our use cases? If seems to me it will always be an objects for our helpers?

Reply:
I might have the type wrong as I had this generated by the IDE
But doesn't this correspond to an object whose keys are string and whose values are string | number | boolean | string[] | number[] ?

Reply:
ohh I see, I misread the types. It makes sense for property fields for parameters, but do we need `null` if we deem it to be invalid in the code below? Do we need to validate the type of parameters here if we already declare Typescript as such `parameters: {[s: string]: string | number | boolean | string[] | number[]}`? 🤔

Reply:
Let's add a small unit test for this